curriculum/challenges/english/25-front-end-development/lecture-working-with-common-string-methods/67326c29dcd98fc5ecc49779.md

```diff
@@ -8,7 +8,68 @@ dashedName: how-can-you-replace-parts-of-a-string-with-another
 
 # --description--
 
-Watch the lecture video and answer the questions below.
+Watch the video or read the transcript and answer the questions below.
+
+# --transcript--
+
+How can you replace parts of a string with another?
+
+In JavaScript, there are many scenarios where you may need to replace a portion of a string with another string.
+
+For instance, you might need to update user information in a URL, change the formatting of dates, or correct errors in user-generated content.
+
+The `replace()` method in JavaScript allows you to find a specified value (like a word or character) in a string and replace it with another value.
+
+Here is the basic syntax:
+
+```js
+string.replace(searchValue, newValue);
+```
+
+`searchValue` is the value you want to search for in the string. It can be either a string or a regular expression (regex), which describes patterns in text. This allows you to search for and manipulate strings in a flexible and powerful way. You’ll learn more about regular expressions in future lecture videos.
```

Comment:
```suggestion
`searchValue` is the value you want to search for in the string. It can be either a string or a regular expression (regex), which describes patterns in text. This allows you to search for and manipulate strings in a flexible and powerful way. You'll learn more about regular expressions in future lecture videos.
```